instances.go

```diff
@@ -16,22 +16,43 @@ package kp
 
 import (
 	"context"
+	"encoding/json"
```

Comment:
There is a typo on lines 290 and 296.

Old: dulaAuthPolicy

New: `dual`AuthPolicy